--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -306,9 +306,9 @@ ExternalProject_Add(brotli
 # In addition, guard these options with -Wno-unknown-warning-option to
 # prevent clang from failing on them.
 set(_boringssl_c_flags "${CMAKE_C_FLAGS}")
-string(APPEND _boringssl_c_flags " -Wno-unknown-warning-option -Wno-stringop-overflow -Wno-array-bounds -Wno-macro-redefined")
+string(APPEND _boringssl_c_flags " -Wno-unknown-warning-option -Wno-stringop-overflow -Wno-array-bounds -Wno-macro-redefined -Wno-error=attributes")
 set(_boringssl_cxx_flags "${CMAKE_CXX_FLAGS}")
-string(APPEND _boringssl_cxx_flags " -Wno-macro-redefined")
+string(APPEND _boringssl_cxx_flags " -Wno-macro-redefined -Wno-error=attributes")

 set(_boringssl_cmake_args
   ${_common_cmake_args}
